packages/odyssey-storybook/.storybook/preview.ts

```diff
@@ -48,6 +48,12 @@ const preview: Preview = {
       source: {
         excludeDecorators: true,
       },
+      argTypes: {
+        sort: "requiredFirst",
```

Comment:
This ensures that args in the `ArgsTable` are displayed alphabetically, with required args first.

Reply:
I'm thinking just alphabetical is fine.

Having them split makes it hard to find things. If there was a sort function, that could be useful, but typically, alphabetical is the easiest for looking up prop names.